index.js

```diff
@@ -126,49 +122,57 @@ open.addEventListener('click', () => {
         </ul>
     </div>
     <div >
-    <img src="image/nature.png" alt="tonic-project" class="image"/>
+    <img src="${projects[position].images}" alt="tonic-project" class="image"/>
     </div>
     <div class="project-details-pop">
         <div class="about-project">
             <p>
-                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            ${projects[position].description}
             </p>
         </div>
         <div class="detailed-popup">
         <div class="technology">
             <ul>
               <li>
-                <p>html</p>
+                <p>${projects[position].language[0]}</p>
               </li>
                <li>
                  <p>
-                    css
+                 ${projects[position].language[1]}
                  </p>
                 </li>
                 <li>
                  <p>
-                    javascript
+                 ${projects[position].language[2]}
                  </p>
                </li>
             </ul>
          </div>
       <div class ="buttons">
-        <button type="submit" class="project1" >
-          See live<i class="fa fa-github"></i>
-        </button>
-        <button type="submit" class="project1" >
-          See source<i class="fa fa-github"></i>
-        </button>
+        <a href="${projects[position].liveLink}"><button type="submit" >
+        See Live<i class="fa fa-github"></i>
+        </button></a>
+        <a href="${projects[position].sourceLink}" target="_blank>
+        <button type="submit" class="source" >See source
+        <i class="fa fa-github"></i>
+        </button></a>
       </div>
     </div>
     </div>
   </div>`;
+  popup.id = 'project-pop';
+  works.appendChild(popup);
+  document.body.style.overflowY = 'hidden';
+}
 
-  main.appendChild(pop);
-  works.appendChild(main);
-  const closePopup = document.querySelector('.close');
-  closePopup.addEventListener('click', () => {
-    main.remove(pop);
+projectButtons.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    addPopup(index);
+    const pophtml = document.getElementById('project-pop');
+    const closePopup = document.querySelector('.close');
+    closePopup.addEventListener('click', () => {
```

Comment:
The popup is responsive, but it does not fit the mobile sizes properly. Kindly make sure that the popup is centered on the page and has the appropriate margins as in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your design:
![image](https://user-images.githubusercontent.com/38088818/174440245-813645c2-8f69-4cf4-9118-c5ec555ce3a2.png)

Figma:
![image](https://user-images.githubusercontent.com/38088818/174440257-144a250f-1c28-4204-b42f-ee5694256c9a.png)